src/Messages.php

```diff
@@ -67,18 +71,23 @@ private function add_hooks() {
 	 * @since 1.1.0
 	 *
 	 * @param string $message_id message identifier
+	 *
+	 * @return bool
 	 */
 	public static function enable_message( $message_id ) {
 
 		if ( ! is_string( $message_id ) || self::is_message_enabled( $message_id ) ) {
 
-			return;
+			return false;
 		}
 
 		$enabled_messages   = self::get_enabled_messages();
 		$enabled_messages[] = $message_id;
 
-		update_user_meta( get_current_user_id(), self::META_KEY_ENABLED_MESSAGES, $enabled_messages );
+		$meta_updated = update_user_meta( get_current_user_id(), self::META_KEY_ENABLED_MESSAGES, $enabled_messages );
+
+		return false !== $meta_updated;
+
```

Comment:
```suggestion
		return (bool) update_user_meta( get_current_user_id(), self::META_KEY_ENABLED_MESSAGES, $enabled_messages );
```

this would suffice, because you have either a number (id of the meta inserted), true, or false; number cast to bool is true